myst_nbs/gaussian_processes/GP-MaunaLoa.myst.md

```diff
@@ -6,26 +6,33 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: pymc-dev
+  display_name: Python 3
   language: python
-  name: pymc-dev
+  name: python3
+substitutions:
+  extra_dependencies: bokeh
 ---
 
-# Example: CO2 at Mauna Loa
+(GP-MaunaLoa)=
+# Gaussian Process for CO2 at Mauna Loa
+
+:::{post} April, 2022
+:tags: gaussian process, CO2
+:category: intermediate
+:author: Bill Engels
```

Comment:
```suggestion
:author: Bill Engels, Chris Fonnesbeck
```

Note: do the changes locally on the notebook, I comment here because I find it better to review.